sessions/js-modern-syntax/destructuring-arrays/index.html

```diff
@@ -3,17 +3,18 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" href="./public/favicon.png" type="image/x-icon" />
```

Comment:
for png you should declare the correct type: `type="image/png"`
```suggestion
    <link rel="shortcut icon" href="./public/favicon.png" type="image/png" />
```